datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -84,12 +84,12 @@ pywin32==306; sys_platform == 'win32' and python_version > '3.0'
 pyyaml==5.4.1; python_version < '3.0'
 pyyaml==6.0.1; python_version > '3.0'
 redis==3.5.3; python_version < '3.0'
-redis==5.0.1; python_version > '3.0'
+redis==5.0.3; python_version > '3.0'
 requests-kerberos==0.12.0; python_version < '3.0'
 requests-kerberos==0.14.0; python_version > '3.0'
 requests-ntlm==1.1.0; python_version < '3.0'
 requests-ntlm==1.2.0; python_version > '3.0'
-requests-oauthlib==1.3.1; python_version < '3.0'
+requests-oauthlib==1.4.0; python_version < '3.0'
```

Comment:
Hmm, we shouldn't do this bump. I'm wondering now how to catch things like this...

The point is that the agent build will succeed, but the wheel is broken, so our py2 tests should fail. Does that mean we need to run those on these PRs?